pkg/operator/ceph/cluster/mon/mon.go

```diff
@@ -194,6 +198,15 @@ func (c *Cluster) Start(clusterInfo *cephclient.ClusterInfo, rookVersion string,
 
 	logger.Infof("targeting the mon count %d", c.spec.Mon.Count)
 
+	monsInDebug, err := c.getMonsBeingDebugged()
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to check for mons in debug")
+	}
+	if monsInDebug.Len() > 0 {
```

Comment:
If a single mon is in debug mode, it will not reconcile on other mons too by this,

But why?

Reply:
If any mon is down, it really can impact the health of other mons, or perhaps mon quorum will even be down. Just to avoid the possibility of mon health being affected by reconciling one mon but not others, I think it's best if all mon reconcile and health checks are skipped if any mon is in debug mode.